### PR TITLE
789-filters-behaviour

### PR DESCRIPTION
**Pull request by chris-lawton**

### CODE REVIEW

**Comment by chris-lawton** [comment]
Hi @hippogriffic - would you be able to review this for me please? Thanks!

File: opentech/apply/funds/templates/funds/base_submissions_table.html [hunk 1]
```diff
@@ -8,25 +8,29 @@
 
 {% block content %}
     {% block table %}
+    <div class="wrapper wrapper--table-actions">
         {% if table.data or active_filters %}
-        <div class="button button--filters button--contains-icons js-open-filters">Filter By</div>
+            <div class="button button--filters button--contains-icons js-toggle-filters">Filters</div>
```
Comment (hippogriffic) [inline opentech/apply/funds/templates/funds/base_submissions_table.html]: is there a reason you're going with a `div` and not `button`?

File: opentech/apply/funds/templates/funds/base_submissions_table.html [hunk 2]
```diff
@@ -8,25 +8,29 @@
...
-        <div class="filters js-filter-wrapper">
-            <div class="filters__header">
-                <div class="js-clear-filters">Clear</div>
-                <div>Filter by</div>
-                <div class="js-close-filters">Close</div>
-            </div>
+        {% if table.data or active_filters %}
+            <div class="filters">
+                <div class="filters__header">
+                    <div class="js-clear-filters">Clear</div>
```
Comment (hippogriffic) [inline opentech/apply/funds/templates/funds/base_submissions_table.html]: same here for `clear` and `close`, seems like they could be `button` elements

File: opentech/apply/funds/templates/funds/base_submissions_table.html (hunk 1 above)
Comment (chris-lawton) [reply opentech/apply/funds/templates/funds/base_submissions_table.html]: Think this was from the first phase of work, but agree that they should all be buttons. Will swap.

**Pull request merged**